docs/home/index.md

```diff
@@ -0,0 +1,304 @@
+---
+orphan: true
+---
+
+
+<style>
+/* Cards with links */
+.sd-hide-link-text {
+  height: 0;
+}
+</style>
+
+
+
+# Welcome to CrateDB!
+
+CrateDB is designed to handle your demanding analytics and search workloads with
+ease. Here you'll find comprehensive resources including reference manuals,
+tutorials, example code, and more.
+
+Happy learning and building with CrateDB!
+
+::::{grid} 1
+:margin: 1
+:padding: 2
+
+:::{grid-item-card} {material-outlined}`rocket_launch;1.7em` CrateDB Cloud
+:link: cloud-docs-index
+:link-type: ref
+:link-alt: CrateDB Cloud
+:padding: 2
+:class-title: sd-fs-5
+
+Discover the fully managed, scalable SQL database designed for
+real-time search and analytics. It supports fast analytics applications with
+real-time indexing of time-series, JSON, vector, text, and geo data. Perform
+near real-time analytics on vast data sets with complex queries and streamline
+data management across multiple cloud providers.
+
+[Get started now!](https://cratedb.com/docs/cloud/tutorials/quick-start.html)
+:::
+
+::::
+
+
+::::{grid} 1 1 2 2
+:margin: 1
+:padding: 2
+
+:::{grid-item-card} {material-outlined}`auto_stories;1.7em` Reference Manual
+:link: cloud-docs-index
```

Comment:
Probably a pasting missclick, this points to cloud

Reply:
Thanks for spotting. Fixed!